File: hw/quiroga/U2/HW_18_Inheritance/Inheritance_JS/javascript/ec/edu/espe/zoo/model/Vaccine.mjs
export class Vaccine {
    #id;
    #description;

    constructor(id, description) {
        this.#id = id;
        this.#description = description;
    }

    toString() {
        return `Vaccine {id: ${this.#id}, description: ${this.#description}}`;
    }

    getId() {
        return this.#id;
    }

    setId(id) {
        this.#id = id;
    }

    getDescription() {
        return this.#description;
    }

    setDescription(description) {
        this.#description = description;
    }
}

File: ws/pilatasig/u2/WS_15_Inheritances/ZooApp/src/main/java/ec/edu/espe/zooapp/model/Vaccine.java

package ec.edu.espe.zooapp.model;

/**
 *
 * @author David Pilatasig
 */
public class Vaccine {
    private int id;
    private String description;

    public Vaccine(int id, String description) {
        this.id = id;
        this.description = description;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return "Vaccine{" + "id=" + id + ", description=" + description + '}';
    }
    
    
}
